bin/start-ccd-web.sh

```diff
@@ -44,11 +45,30 @@ az acr login --name hmcts --subscription ${SUBSCRIPTION_ID}
 # Compose CCD Web
 #####################################################################################
 
-docker-compose -f docker-compose.yml up ${@} -d ccd-case-management-web \
-                                                dm-store \
-                                                ccd-api-gateway \
-                                                idam-api \
-                                                authentication-web \
-                                                smtp-server \
-                                                ccd-importer \
-                                                idam-importer
+echo "Starting docker containers..."
+
+docker-compose up -d
+
+while [[ `docker ps -a | grep starting | wc -l | awk '{$1=$1};1'` != "0" ]]
+do
+    echo "Waiting for "`docker ps -a | grep starting | wc -l | awk '{$1=$1};1'`" container(s) to start. Sleeping for 5 seconds..."
+    sleep 5
+done
+
+# for convenience, let's have personal caseworker account
+
+curl -XPOST \
```

Comment:
Will this work if you run `star-ccd-web.sh` after is being executed once ? I assume it will throw an error that user already exists ?

Reply:
I would say these could go into separate scripts and include instructions in readme to execute those whenever required

Reply:
```bash
$ ./bin/start-ccd-web.sh 
    "name": "Donatas.Martinkus@HMCTS.NET",
You are logged into HMCTS Azure Portal
Logging into the HMCTS Azure Container Registry
Login Succeeded
Starting docker containers...
bulk-scan-shared-infrastructure_smtp-server_1 is up-to-date
bulk-scan-shared-infrastructure_service-auth-provider-api_1 is up-to-date
bulk-scan-shared-infrastructure_ccd-api-gateway_1 is up-to-date
bulk-scan-shared-infrastructure_shared-database-v10_1 is up-to-date
bulk-scan-shared-infrastructure_idam-api_1 is up-to-date
bulk-scan-shared-infrastructure_ccd-user-profile-api_1 is up-to-date
Starting bulk-scan-shared-infrastructure_ccd-importer_1 ... 
Starting bulk-scan-shared-infrastructure_ccd-importer_1       ... done
Starting bulk-scan-shared-infrastructure_dm-store_1                ... done
Starting bulk-scan-shared-infrastructure_authentication-web_1 ... done
Starting bulk-scan-shared-infrastructure_idam-importer_1      ... done
Starting bulk-scan-shared-infrastructure_ccd-case-management-web_1 ... done
Starting bulk-scan-shared-infrastructure_ccd-data-store-api_1      ... done
Waiting for 1 container(s) to start. Sleeping for 5 seconds...
Waiting for 1 container(s) to start. Sleeping for 5 seconds...
... some waits here
Waiting for 1 container(s) to start. Sleeping for 5 seconds...
Waiting for 1 container(s) to start. Sleeping for 5 seconds...
User with this email account already exists
```
and script ends

Reply:
ok that's fine still separating them gives ability to create it separately so bonus :)

Reply:
bonuses are my thing, right? 🙂